src/main/java/com/trzewik/blackjack/game/Game.java

```diff
@@ -103,7 +106,7 @@ private void playersAuction(){
                 if (player.getHand().size() >= 11){
                     player.setLastMove(MoveType.STAND);
                 }
-                if (player.getLastMove() != MoveType.STAND && player.getLastMove() != MoveType.DOUBLEDOWN){
+                if (player.getLastMove() != MoveType.STAND && player.getLastMove() != MoveType.DOUBLE_DOWN){
```

Comment:
jesli nie jest tym i nie jest tym, to jest czym? 😄  w tym wypadku chyba lepiej bedzie zrobic logike, ze jesli cos jest tym, to zrob to...

Reply:
tak, tylko wtedy to sie naturalniej czyta, nie uważasz?